src/test/java/guitests/UITest.java

```diff
@@ -362,6 +362,29 @@ private void awaitCondition(Callable<Boolean> condition, int timeoutInSeconds) {
     }
 
     /**
+     * Performs logout from File -> Logout on HubTurbo's pView.
+     */
+    public void logout(){
+        clickMenu("File", "Logout");
+    }
+
+    /**
+     * Performs UI login on the login dialog box.
+     * @param owner
+     * @param repoName
+     * @param username
+     * @param password
```

Comment:
Yup, it is useful. The @param is actually auto-generated by Intellij (which performs this according to Javadoc standards). The documentation appears like this - 
``java
guitests.UITest
public void login(String owner,
                  String repoName,
                  String username,
                  String password)
Performs UI login on the login dialog box.
Parameters:
owner - The owner of the repo.
repoName - The repository name
username - The Github/dummy username
password - The Github/dummy password
``